fff

```diff
@@ -141,7 +141,7 @@ get_ls_colors() {
 
 get_w3m_path() {
     # Find the path to the w3m-img library.
-    w3m_paths=(/usr/{local/,}{lib,libexec,lib64,libexec64}/w3m/w3mi*)
+    w3m_paths=(/usr/{pkg/,}{local/,}{bin,lib,libexec,lib64,libexec64}/w3m/w3mi*)
```

Comment:
Can this be adjusted to just an environment variable or a configuration variable that could be overriden via Makefile?

I.e. that would be still problematic on NetBSD and other possible pkgsrc platforms - and probably also other package systems - where the PREFIX (e.g. `/usr/pkg`) is configurable.

Thanks!